src/AccessibilityInsights.Extensions.AzureDevOps/Models/IssueResult.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+using System;
+
+namespace AccessibilityInsights.Extensions.AzureDevOps.Models
+{
+    public class IssueResult : IIssueResult
```

Comment:
>IssueResult [](start = 17, length = 11)

Please add some /// comments to clarify the role of this class and its methods

Reply:
Commented